docs/generated/packages/js.json

```diff
@@ -314,6 +314,27 @@
             "description": "When `updateBuildableProjectDepsInPackageJson` is `true`, this adds dependencies to either `peerDependencies` or `dependencies`.",
             "enum": ["dependencies", "peerDependencies"],
             "default": "peerDependencies"
+          },
+          "external": {
+            "oneOf": [
+              {
+                "type": "string",
+                "description": "Mark which other projects to be external to the project. Eg: projects need to be published and installed separately",
+                "enum": ["all", "none"],
+                "default": "all"
+              },
+              {
+                "type": "array",
+                "items": { "type": "string" },
+                "description": "Mark which other projects to be external to the project. Eg: projects need to be published and installed separately"
+              }
+            ]
+          },
```

Comment:
This will probably be changed based on what we discussed on buildable mixing with non-buildable